projects/storefrontapp-e2e-cypress/cypress/helpers/checkout-forms.ts

```diff
@@ -126,6 +126,6 @@ export function fillPaymentDetails(
 
     cy.get('button.btn.btn-block.btn-primary')
       .contains('Continue')
-      .click({ force: true });
```

Comment:
`{ force: true }` Failes the ASM e2e test checkout.  Forcing the click on a disabled submit button seems to be a guaranteed timeout in this context. By default, Cypress waits for the button to be enabled before clicking.